Improve Logical Flow of Documentation (#324)
hyperspy/rosettasciio: doc/index.rst

microscopy community, data formats used by this community are still particularly
well represented.

.. grid:: 2 3 3 3
  :gutter: 2

  .. grid-item-card::
    :link: user_guide/install
    :link-type: doc

    :octicon:`rocket;2em;sd-text-info` Getting Started
    ^^^

    New to rosettasciio or Python? The getting started guide provides an
    introduction on basic usage of rosettasciio and how to install it.

  .. grid-item-card::
    :link: user_guide/supported_formats/index
    :link-type: doc

    :octicon:`checklist;2em;sd-text-info` Supported Formats
    ^^^

    This provides a nice overview of the different files that can be read and written by rosettasciio.

  .. grid-item-card::
    :link: user_guide/interoperability
    :link-type: doc

    :octicon:`arrow-switch;2em;sd-text-info` Interoperability
    ^^^

    Documentation describing how to use rosettasciio with other libraries than HyperSpy.

  .. grid-item-card::
    :link: file_specification/index
    :link-type: doc

    :octicon:`checklist;2em;sd-text-info` File Specifications
    ^^^

>     Links to File Specifications for the different file formats supported by rosettasciio.

Reviewer: ```suggestion
    Links to or documentation of file specifications for some of the formats supported by RosettaSciIO.
```

  .. grid-item-card::
    :link: api/index
    :link-type: doc

    :octicon:`book;2em;sd-text-info` API Reference
    ^^^

    Documentation of the Application Programming Interface (API),

  .. grid-item-card::
    :link: contributing
    :link-type: doc

    :octicon:`code-square;2em;sd-text-info` Contributing
    ^^^

>     Information on how to contribute to the development of rosettasciio.

Reviewer: ```suggestion
    Information on how to implement new file plugins or help improve existing ones.
```

RosettaSciIO provides the dataset, its axes and related metadata contained in a

> file in a python dictionary that can be easily handled by other libraries.

Reviewer: ```suggestion
file in a :ref:`python dictionary <interfacing-api>` that can be easily handled by other libraries.
```

microscopy community, data formats used by this community are still particularly
well represented.

.. grid:: 2 3 3 3
  :gutter: 2

  .. grid-item-card::
    :link: user_guide/install
    :link-type: doc

    :octicon:`rocket;2em;sd-text-info` Getting Started
    ^^^

    New to RosettaSciIO or Python? Find information on how to
    install RosettaSciIO.

  .. grid-item-card::
    :link: supported_formats/index
    :link-type: doc

    :octicon:`checklist;2em;sd-text-info` Supported Formats
    ^^^

    Overview of the different file formats that RosettaSciIO can read
    from and write to, as well as information on specific options if applicable.

  .. grid-item-card::
    :link: user_guide/interoperability
    :link-type: doc

    :octicon:`arrow-switch;2em;sd-text-info` Interoperability
    ^^^

>    Information on scripts and packages that allow opening data saved using
>    RosettaSciIO with third-party programs.

Reviewer: ```suggestion
    Information on scripts and packages that allow opening data saved using
    RosettaSciIO with third-party programs.
```

  .. grid-item-card::
    :link: file_specification/index
    :link-type: doc

    :octicon:`checklist;2em;sd-text-info` File Specifications
    ^^^

>      Links to or documentation of file specifications for
>      some of the formats supported by RosettaSciIO.

Reviewer: ```suggestion
    Links to or documentation of file specifications for
    some of the formats supported by RosettaSciIO.
```

   user_guide/index
   file_specification/index

>    user_guide/supported_formats

Reviewer: Does not work currently, and is already a subcategory of user_guide. Instead add a grid to the main page with a prominent link to the table of supported formats (Maybe even before installation and any other category).

Reviewer: How about a grid with six items:

* Supported Formats
* Installation
* Interoperability

and in a second row:

* File specifications
* API
* Contributing

That could replace the whole TOC and come before the splitting note and the citing section!
